### ISSUE TITLE
Cancel on OnNavigationStarted doesn't work

### ISSUE THREAD

**Issue by manuchevreux** [SKLn-Rad/Xam.Plugin.Webview#63]
Like in sampleApp i use this code on my .xaml
`<fwv:FormsWebView x:Name="Browser" Source="https://mysite" OnNavigationStarted="FormsWebView_OnNavigationStarted"  Grid.Column="0" Grid.Row="0" Grid.ColumnSpan="9" Grid.RowSpan="20"  />`
in my .cs:
`  private void  FormsWebView_OnNavigationStarted(object sender , Xam.Plugin.WebView.Abstractions.Delegates.DecisionHandlerDelegate e)
        {
           e.Cancel = true;
         }`
And when i click on a <a href"mynewpage"></a> My new page load , my e.Cancel does'nt work.

**Comment by SKLn-Rad** [comment]
Is this Android? I did a bug fix which may help. Can you try this build?
https://www.nuget.org/packages/Xam.Plugin.WebView/2.1.3

**Comment by manuchevreux** [comment]
Yeah now it's working but I can't do  e.Cancel = true; Navigation.PushAsync(new PageRegister()); :/
